pkg/pf/internal/check/check_test.go

```diff
@@ -100,7 +100,7 @@ func TestMissingIDProperty(t *testing.T) {
 	})
 
 	assert.Equal(t, "error: Resource test_res has a problem: no \"id\" attribute. "+
-		"To map this resource consider specifying ResourceInfo.ComputeID\n", stderr)
+		"To map this resource consider specifying ResourceInfo.ComputeID to a valid field on the upstream resource\n", stderr)
```

Comment:
Can be multiple fields. I've found many AWS resources being added have composite IDs https://github.com/pulumi/pulumi-aws/blob/master/provider/resources.go#L5860